combat-trainer.lic

```diff
@@ -712,7 +712,7 @@ class SafetyProcess
 
   def execute(game_state)
     custom_require.call('tendme') if bleeding? && !Script.running?('tendme')
-    fput 'exit' if health < 40
+    fput 'exit' if health < get_settings.health_threshold
```

Comment:
It's not as simple as using `get_settings` here because then it won't be using the same settings as the rest of `combat-trainer` (i.e. the script could have been executed with args). You'll want to pass the _right_ settings into `SafetyProcess` (see line 2154).